.travis.yml

```diff
@@ -44,7 +44,7 @@ env:
   - SPREAD_PATH=/tmp/spread
   matrix:
   - SYSTEM=google:ubuntu-devel-64 VARIANT=amd64
-  - SYSTEM=google:fedora-rawhide-64 VARIANT=amd64
+#  - SYSTEM=google:fedora-rawhide-64 VARIANT=amd64 Getting GPG errors from dnf today
```

Comment:
rawhide changed over to Fedora 31 keys last week, so your rawhide VM probably needs rebootstrapping.